lib/hpc/thread1.g

```diff
@@ -20,7 +20,8 @@
 # Mock functions from thread1.g
 
 BIND_GLOBAL("MakeThreadLocal", ID_FUNC);
-BIND_GLOBAL("MakeReadOnlyObj", ID_FUNC);
+# HACK: improve if at some point there exists proper read-only support in GAP
```

Comment:
I find this comment more confusing than helpful. Perhaps it meant to say: "remove this if the read-only support in HPC-GAP is ported to GAP"? But then this is obvious.

So unless you intended some other meaning (in that case: please elaborate),  I'd just remove this comment.

Reply:
No, this is exactly what I meant. I added the comment since I wanted to make clear that this is a hack since read-only objects in HPC-GAP work differently than immutable ones in GAP. That makes more sense in the commit message, so I'll put it there.